src/core/getters/resReqTypes.ts

```diff
@@ -12,6 +12,7 @@ import { resolveObject } from '../resolvers/object';
 import { getRef } from './ref';
 
 const CONTENT_TYPES = [
+  '*/*',
```

Comment:
why do you need this content type?

Reply:
> why do you need this content type?

spring boot can be use it: https://github.com/spring-projects/spring-framework/blob/master/spring-web/src/main/java/org/springframework/http/MediaType.java#L64